infra/monitoring/main.tf

```diff
@@ -58,21 +58,25 @@ data "aws_iam_policy_document" "push_metric_policy" {
 }
 
 resource "aws_iam_policy" "push_metric_policy" {
-  name        = "push-metric-policy"
+  name        = var.module_name == "hudi" ? "push-metric-policy-hudi" : "push-metric-policy-json"
   path        = "/"
   description = "Allow push metrics to CloudWatch"
 
   policy = data.aws_iam_policy_document.push_metric_policy.json
 }
 
+
+
 resource "aws_iam_role" "metric_pusher_lambda_role" {
-  name               = "metric-pusher-lambda-role"
+  #name               = "metric-pusher-lambda-role"
```

Comment:
Do not leave old commented code

Reply:
This applies to multiple places and files

Reply:
commented are left just as im already testing but yeah, will be removed before pushing